Formula/rdiff-backup.rb

```diff
@@ -1,10 +1,9 @@
 class RdiffBackup < Formula
   desc "Reverse differential backup tool, over a network or locally"
   homepage "https://rdiff-backup.net/"
-  url "https://github.com/rdiff-backup/rdiff-backup/releases/download/v2.0.5/rdiff-backup-2.0.5.tar.gz"
-  sha256 "2bb7837b4a9712b6efaebfa7da8ed6348ffcb02fcecff0e19d8fff732e933b87"
+  url "https://github.com/rdiff-backup/rdiff-backup/releases/download/v2.2.0/rdiff-backup-2.2.0.tar.gz"
+  sha256 "951779e051f5f0a1da8c6608e4aa87e60649751c8f291e388f6292577eb1f494"
```

Comment:
May want to consider using PyPI tarball instead to avoid missing dependencies. `rdiff-backup >= 2.1` has introduced a dependency on `pyyaml` (which we could use formula for).

We should also consider letting python manage installation so that the libraries get installed correctly and we can get correct version info - https://github.com/orgs/Homebrew/discussions/4066. Version info is accessed via metadata of library which either needs libraries to be available in default location or inside a venv.